pkgs/tools/inputmethods/gebaar-libinput/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, pkgconfig, cmake, libinput, zlib }:
```

Comment:
Not a big deal, but personally I like these to be at least separated with a blank line before the `mkDerivation`, or even better - put each argument in it's own line.

Reply:
I use nixfmt and leave these things alone.